ClassNLLCriterion.lua

```diff
@@ -1,18 +1,16 @@
-local ClassNLLCriterion, parent = torch.class(
-    'nn.ClassNLLCriterion',
-    'nn.Criterion'
-)
+local THNN = require 'nn.THNN'
+local ClassNLLCriterion, parent = torch.class('nn.ClassNLLCriterion', 'nn.Criterion')
 
 function ClassNLLCriterion:__init(weights, sizeAverage)
     parent.__init(self)
     if sizeAverage ~= nil then
-        self.sizeAverage = sizeAverage
+       self.sizeAverage = sizeAverage
```

Comment:
some of this is 3-indented, and some is 4-indented :-P

I really hate 3-indentation by the way.   2-indentation is quite nice.  4-indentation is quite standard.  3-indentation is one of the things that makes me run away and write python wrappers instead :-P